test-helpers/src/connection/cassandra/cql_ws.rs

```diff
@@ -280,4 +284,26 @@ impl ServerCertVerifier for SkipVerifyHostName {
             Err(err) => Err(err),
         }
     }
+
```

Comment:
Why do we need to add the following new functions here?

Reply:
Its just following the new rustls API.
Refer to https://github.com/shotover/shotover-proxy/pull/1387/files#diff-b37d1d20c43783d212ac312f79ebbba68058679501116dcdf45beda95fe4ea5bR271-R291 where we made the same change in shotover itself.